static/js/components/ProgramCourse.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React from 'react';
+import Popover from 'material-ui/Popover';
+
+
+export default class ProgramCourse extends React.Component {
+  props: {
+    id: number,
+    title: string,
+    description: string,
+    url: string,
+    enrollment_text: string,
+  }
+  state = {
+    isOpen: false,
+  }
```

Comment:
we generally don't use `this.state` much (generally we use redux instead). I think it's fine here, since it's one tiny bit of state and it's outside of the main application.